ansible-tests/validations/pacemaker-status.yaml

```diff
@@ -0,0 +1,20 @@
+---
+- hosts: controller
+  become: true
```

Comment:
Not strictly necessary since tripleo-ansible-inventory already takes care of setting the 'become' parameter. But I guess it doesn't hurt leaving it here.